cpp/open3d/t/geometry/Geometry.h

```diff
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <fmt/format.h>
+
```

Comment:
Would be great to not leak fmt in the headers.
Otherwise you would need to make another PR if fmt breaks / is not updated.
Better to, when possible defines includes / using in .cpp

Reply:
That's a good suggestion, but currently fmt is used throughout Open3D and it will need significant refactoring to prevent header leaks. fmt is included with the Open3D binary package. In the future, we will switch to the upcoming C++20 fmt support.